dashboard/templatetags/custom_tag.py

```diff
@@ -0,0 +1,11 @@
+from urllib.parse import unquote
+from django import template
+
+register = template.Library()
+
+@register.filter
+def append_ast_if_req(field):
+    if field.field.required:
+        return field.label + '*'
+    else:
+        return field.label
```

Comment:
this is the filter that I added that can be used in template. It's used by writing {{ field ! append_ast_if_req }} where field is a field of a form. It returns the field label along with an '*' if it's a required field. The @register registers the filter into the django library so it can be used in template.